### PR TITLE
KH-211: Add spa frontend config url in the start-distro.sh

### PR DESCRIPTION
**Pull request by kdaud**
Refer to [KH-211](https://mekomsolutions.atlassian.net/browse/KH-211)

### CODE REVIEW

File: configs/openmrs_config/attributetypes/attribute_types.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Uuid,Void/Retire,Entity name,Name,display:en,display:km,Description,Min occurs,Max occurs,Datatype classname,Datatype config,Preferred handler classname,Handler config,_order:1000
 6f9f22c3-c3e8-4683-b55b-42943a46d8bb,,Visit,Visit Sequence,Visit Sequence,ប្រភេទជំងឺមកពិនិត្យ,,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,aff366ff-5477-4e45-a795-f79fb2db5c9d,,,
-af89d44d-5074-472f-b5bc-470feb567633,,Visit,Insurance Type,Insurance Type,ប្រភេទធានារ៉ាប់រង,,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,8e807373-be15-4580-9a1c-0d681e04a2ed,,,
-2792fddc-1ca6-4093-84d2-853bd9b0245e,,Visit,Punctuality,Punctuality,មកពិនិត្យទាន់ពេលវេលា,,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype,d81d4698-e78c-420d-aac5-cb1f606fb32e,,,
\ No newline at end of file
+af89d44d-5074-472f-b5bc-470feb567633,,Visit,Insurance Type,Insurance Type,ប្រភេទធានារ៉ាប់រង,,0,1,org.openmrs.customdatatype.datatype.ConceptDatatype.name,8e807373-be15-4580-9a1c-0d681e04a2ed,,,
```
[the remark this answers is not in the document]
Comment (kdaud) [reply configs/openmrs_config/attributetypes/attribute_types.csv]: Specifically the UUIDS for punctuality and Insurance type are being overridden by those from openmrs-distro-referenceapplication

**Comment by kdaud** [comment]
The start visit form is able to load fully.

[visit.webm](https://github.com/openmrs/ozone-distro-cambodia/assets/38831673/c3d9964c-367c-4d8f-8cec-2a531a66c588)

**Comment by kdaud** [comment]
@kazlaw for some reason, the issue this PR is addressing is no longer existing on the dev server. I checked it out early morning and could not reproduce the issue. Could be the server was restarted and things got sorted.

If you get some time try to checkout the behavior of the start visit form when starting a patient visit on the dev server.

**Comment by kazlaw** [comment]
@vasharma05 , mentioned that a PR solving this was merged in O3 ie it was due to a regression

**Comment by kdaud** [comment]
Hi @kazlaw, though the dev server the issue disappeared but when starting kh server via gitpod I notice the start patient visit form does not load fully. When get some time try it out on end via gitpod on the main branch.

**Comment by kdaud** [comment]
Closing this PR since the issue was fixed and is no longer existing.

**Pull request closed without merging**